src/state/zustand/swapperStore/selectors.ts

```diff
@@ -49,9 +49,9 @@ export const selectAvailableSwappersWithMetadata = (state: SwapperState) =>
   state.availableSwappersWithMetadata
 export const selectSelectedCurrencyToUsdRate = (state: SwapperState) =>
   state.selectedCurrencyToUsdRate
-
 export const selectSlippage = (state: SwapperState): string =>
   state.activeSwapperWithMetadata?.quote.recommendedSlippage ?? DEFAULT_SLIPPAGE
+export const selectAffiliateBps = (state: SwapperState): string => state.affiliateBps
```

Comment:
If we remove this one and use `selectSwapperDefaultAffiliateBps` instead, should be the same guarantees?

Reply:
disagree - `selectSwapperDefaultAffiliateBps` is the bps to be applied to a given swapper if the user has checked donations on, this is for the actively selected amount. 

perhaps this could be named `selectActiveAffiliateBps` or `selectChosenAffiliateBps` though?

Reply:
Yeah, this is the default bps for a given swapper, which is precisely why I would rather rename it `selectSwapperAffiliateBps` and remove the `selectAffiliateBps` - the selector itself is effectively a `getOrElse` (else being if no swapper is selected) '0'.
The action really only calls a selector and sets the zustand field from it, which `selectAffiliateBps` then selects - which is effectively overhead vs. simply consuming a selector.

re: `this is for the actively selected amount.` it's actually static:

https://github.com/shapeshift/web/blob/2b2f291c5413ca99f264dbd1693b25e91de26075/src/state/zustand/swapperStore/selectors.ts#L86-L105

Reply:
Not quite, I'll explain, and add some context re: the design decisions.

`affiliateBps` is the _active_ (good name suggestion @0xdef1cafe, I'll update accordingly) bps, which at the moment can only be `0` or the THORSwapper default of `30`.

The design accommodates part two of this feature, which is to have user selectable bps of arbitrary n between some range (e.g. 0 to 1000). It anticipates this by not coupling the default swapper bps (accessed via `selectSwapperDefaultAffiliateBps`) and `affiliateBps`, as the swapper default is just a static value.

Reply:
Updated `affiliateBps` to `activeAffiliateBps` in https://github.com/shapeshift/web/pull/4353/commits/5e4ab8cec7decef9409b801a6949860faece12c1.

Reply:
Fair enough - if there's going to be non-arbitrary bps support, then this architecture definitely makes sense.